rust/ql/lib/codeql/rust/controlflow/internal/ControlFlowGraphImplSpecific.qll

```diff
@@ -0,0 +1,63 @@
+import codeql.controlflow.Cfg
+import rust as Rust
+private import SuccessorType as ST
+private import Scope as Scope
+
+module CfgInput implements InputSig<Rust::Location> {
```

Comment:
I would just move this into `ControlFlowGraphImpl.qll` and get rid of `ControlFlowGraphImplSpecific.qll`.

Reply:
I picked this separation up from `swift`, but yeah I agree that it doesn't seem to meaningful.